pkg/cmd/cli/describe/describer.go

```diff
@@ -640,7 +640,9 @@ func (d *ImageStreamDescriber) Describe(namespace, name string, settings kctl.De
 
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, imageStream.ObjectMeta)
-		formatString(out, "Docker Pull Spec", imageStream.Status.DockerImageRepository)
+		if len(imageStream.Spec.DockerImageRepository) > 0 {
+			formatString(out, "Imported From", imageStream.Spec.DockerImageRepository)
```

Comment:
why "Imported From" ? I thought this field contains the docker pull spec (so I can docker pull <image>).